### DISCUSSION TITLE
Error about "PETSC LAPACK syev routine returned error code 2"

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#22641] [Q&A Modules: Solid mechanics] [answered]
Hi Moose experts or users,

I got the following errors. Any ideas to solve that? Thank you
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/74226455/200903588-f0f98658-ccb7-4641-985f-ddd03a1602e1.png">

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you please attach the full simulation log?
Also do you know which tensor that was? If you could paste it here that could help

Guillaume

**Reply by xchengood** [reply]
Here is the error log. I have no any ideas about which tensor that was. Thank you.
[error log.txt](https://github.com/idaholab/moose/files/9974853/error.log.txt)

**Reply by GiudGiud** [reply]
@sapitts  could you please help with this

**Reply by hugary1995** [reply]
This error comes from eigvalue decomposition of the right cauchy green strain, for the RU decomposition. You can go to the corresponding frame in a debugger, then print the failing tensor. Although unlikely, if lapack fails to find eigenvalues and eigenvectors of a well-conditioned symmetric r2t, we can attempt to fix that.
